third-party/qthread/qthread-1.10/config/config.sub

```diff
@@ -238,6 +238,7 @@ case $basic_machine in
 	# Some are omitted here because they have special meanings below.
 	1750a | 580 \
 	| a29k \
+	| aarch64 | aarch64_be \
```

Comment:
Right, I'd like to avoid changing lower-level stuff in third-party.  I discussed this with @ronawho and something will be done to encourage the qthreads folks to update their autoconf.  For now, this manual change is necessary, but it will become obsolete when qthreads updates to a later autoconf.